modules/m_register_api/api_session.cpp

```diff
@@ -87,7 +87,10 @@ SessionRef Session::Find(const Anope::string& id, bool touch, bool check)
 		return NULL;
 
 	if (touch)
+	{
 		sess->lastused = Anope::CurTime;
+		sess->QueueUpdate();
```

Comment:
`QueueUpdate` appears to be from Anope's `Serializable` class.

Reply:
`Session` inherits from `Serializable`. Queueing the update here is used to mark it as needing to be written to the SQL database.